README.md

```diff
@@ -21,6 +21,567 @@ COPY --from=terraform-root-modules /aws/ /conf/
 WORKDIR /conf/
 ```
 
+Then provision the Terraform modules required for the project.
+
+
+## Cold Start
```

Comment:
I strongly prefer we move this documentation to our docs portal. 

1. Avoid fragmentation -> technical debt & maintenance problems
2. Validation
3. Consolidation of github issues
4. All the other benefits of our portal (e.g. searchable)

Reply:
yes will move it to the docs portal and add references here